src/main/scala/is/hail/expr/Relational.scala

```diff
@@ -344,6 +344,83 @@ case class MatrixRead(
   override def toString: String = s"MatrixRead($path, dropSamples = $dropCols, dropVariants = $dropRows)"
 }
 
+case class MatrixRange(nRows: Int, nCols: Int, nPartitions: Int) extends MatrixIR {
```

Comment:
should we let users specify the row and column names like we do with table?

Reply:
Table wasn't actually  exposed in Python.  I hardcoded the names everywhere.